cypress/integration/anton.sample/todoSample.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Managing of todo list
+  As a user
+  I want to use todo list functionality
+  So I can plan my activities
+
+  Background:
+    When I add items via localStorage
+    When I open todo list page
+    Then I see that header is shown
+    Then I see that "item1" item is presented
+    Then I see that footer is shown
+
+  Scenario: Add new item into the list
+    When I type "Test1" into input field and press Enter
+    Then I see that "Test1" item is presented
+    Then I see that number of left items is "4"
+
+  Scenario: Edit added item from the list
+    When I double click on "item1" item
```

Comment:
Same here. When `I change "item1" item to "Test1"` instead of this and next steps.

Reply:
And don't forget you can wrap simple element interactions in meaningful page methods. `page.editTodo('OldName', 'NewName')`